gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/GapicPackageInfo.java

```diff
@@ -20,8 +20,16 @@
 /** A thin wrapper around PackageInfoDefinition to maintain a clean separation of concerns. */
 @AutoValue
 public abstract class GapicPackageInfo {
+
+  public static final GapicPackageInfo EMPTY =
+      builder().setPackageInfo(PackageInfoDefinition.EMPTY).build();
+
   public abstract PackageInfoDefinition packageInfo();
 
+  public boolean isEmpty() {
+    return packageInfo().shouldGenerateFile();
+  }
```

Comment:
nit: I see from the comment above that this is `A thin wrapper around PackageInfoDefinition`. If it makes sense, perhaps we can name it the same?

GapicPackageInfo's shouldGenerateFile() which calls PackageInfoDefinition's shouldGenerateFile()

Reply:
As mentioned above, changes to package info are being dropped in favor of a no-op at the `Writer` level